### PR TITLE
Janezd kmeans gui changes

### PR DESCRIPTION
**Pull request by PrimozGodec**
This pull request is merged pull request https://github.com/PrimozGodec/orange3-educational/pull/1 with the latest version of Interactive k-means.

### CODE REVIEW

**Comment by PrimozGodec** [comment]
I also merged changes from https://github.com/PrimozGodec/orange3-educational/pull/2 pull request by @janezd and shorten code line length to maximal width 80 characters.

It is now ready to merge.

**Comment by PrimozGodec** [comment]
Additionally I fixed bug with nan values in data set. 

It is ready to merge.

File: orangecontrib/educational/widgets/owkmeans.py [hunk 1]
```diff
@@ -1,15 +1,19 @@
+import numpy as np
+from os import path
+from itertools import chain
+import time
+
+from PyQt4.QtCore import pyqtSlot, QThread, SIGNAL, Qt
```
Comment (kernc) [inline orangecontrib/educational/widgets/owkmeans.py]: pyqtSignal. http://pyqt.sourceforge.net/Docs/PyQt4/new_style_signals_slots.html

Comment (PrimozGodec) [inline orangecontrib/educational/widgets/owkmeans.py]: Thanks kernc. I chaged it.

**Pull request merged**